src/modules/users/dto/follow.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+import { IsUUID } from 'class-validator';
+
+export class FollowVendorDto {
+  @ApiProperty({
+    example: '2c674384-f944-401b-949b-b76e8793bdc2',
+    description: 'The ID of the buyer',
+  })
+  @IsUUID()
+  buyerId: string;
```

Comment:
but you can get this Id from userToken

Reply:
fixed here https://github.com/ZenBit-Tech/code-lions_be/pull/35/commits/10f2ee4ae7b09cbeb4a612b35e530954e3fef8a2